### PR TITLE
tama na

### PR DESCRIPTION
**Pull request by MostDeadDeveloper**
Reviewing Code Exercise #1

### CODE REVIEW

File: Mic-Python-CodeExercises/exer1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import time
```
Comment (MostDeadDeveloper) [inline Mic-Python-CodeExercises/exer1.py:1]: Next time , Do the work in the actual  Problem1.py and Problem2.py files. 

Git  can track the changes of all the files in the folder you are using, dont worry. That is the power of git.

Comment (MostDeadDeveloper) [reply Mic-Python-CodeExercises/exer1.py:1]: rEVIEWING

**Review by MostDeadDeveloper — COMMENTED** [review]
Hmm

File: Mic-Python-CodeExercises/exer1.py (hunk 1 above)
Comment (MostDeadDeveloper) [reply Mic-Python-CodeExercises/exer1.py:1]: Type here if you see the changes

Comment (MostDeadDeveloper) [reply Mic-Python-CodeExercises/exer1.py:1]: Presss the "resolve conversation" if you have read this part..

Comment (Mica56) [reply Mic-Python-CodeExercises/exer1.py:1]: ahh okii kanina pa ako naliligaw kung san ba hahanapin yung comment mo xD

**Comment by MostDeadDeveloper** [comment]
Changes

**Review by Mica56 — APPROVED** [review]

**Pull request closed without merging**